resources/metadata/sonar.yaml

```diff
@@ -65,6 +65,20 @@ spec:
         resourceRef:
           - name: commonPipelineEnvironment
             param: artifactVersion
+      - name: projectKey
```

Comment:
step extension could be a change on it's own

Reply:
Ok, will do. My current status is that I have not yet managed to generate coverage reports in my test project that the sonarQube scan will pick up properly.